events/wc_druidic_events.txt

```diff
@@ -115,9 +148,33 @@ character_event = {
 narrative_event = {
 	id = WCDRU.300
 	title = EVTTITLE_WCDRU_300
-	desc = EVTDESC_WCDRU_300
 	picture = GFX_evt_emerald_dream
 	border = GFX_event_narrative_frame_war
+	
+	desc = {
+		trigger = {
+			OR = { 
+				NOT = { has_dlc = "Mystics" }
+				ROOT = {
+					has_artifact = book_of_ur 
+					any_courtier_or_vassal = { has_artifact = book_of_ur }
+				}
+			}
+		}
+	text = EVTDESC_WCDRU_300.1
+	}
+	desc = {
+		trigger = {
+			OR = { 
+				NOT = { has_dlc = "Mystics" }
+				ROOT = {
```

Comment:
I believe something like this can be used `event_target:target_mage = { has_artifact = scythe_of_elune }` instead of checking ROOT and all courtiers and vassals.